layout/column.ftd

```diff
@@ -17,6 +17,12 @@ integer border-bottom: $ds.borders.none
 integer border-left: $ds.borders.none
 integer border-right: $ds.borders.none
 optional ftd.color border-color:
+optional ftd.anchor anchor:
+optional integer top:
+optional integer bottom:
+optional integer left:
+optional integer right:
+optional integer z-index:
```

Comment:
Floater should also put transparency on remaining content, eg if I add a `ds.floater`, it will add an element with transparency that will cover the entire window.